src/Simplex/Chat/Messages.hs

```diff
@@ -790,7 +831,7 @@ data PendingGroupMessage = PendingGroupMessage
 
 type MessageId = Int64
 
-data ConnOrGroupId = ConnectionId Int64 | GroupId Int64
+data ConnOrGroupId = ConnectionId Int64 | GroupId Int64 | NoteFolderId Int64
```

Comment:
this is only used in "deliver to connection" functions and maybe some other "service messages" related functions, so it doesn't need a new constructor